molten-trace/src/main/java/com/hotels/molten/trace/MoltenTrace.java

```diff
@@ -56,7 +56,7 @@ public static void initialize() {
      */
     public static void initialize(boolean onEachOperator) {
         uninitialize();
-        LOG.info("Integrating tracing with Molten...");
+        LOG.info("Integrating tracing with Molten onEachOperator={}", onEachOperator);
```

Comment:
Please consider adding this to the other `Molten*.initialize(boolean onEachOperator)` methods

Reply:
Done in fd3e6c3